### PR TITLE
edav_info_heatmap_korean

### PR DESCRIPTION
**Pull request by juliajungahshin**
edav_info_heatmap_korean

### CODE REVIEW

File: edav_info_heatmap_korean.Rmd [hunk 1]
```diff
@@ -0,0 +1,173 @@
+# edav.info Heatmap in Korean 
+
+Ji In Choi and Jung Ah Shin
+
+```{r setup, include=FALSE}
```
Comment (st3186) [inline edav_info_heatmap_korean.Rmd]: this should read ```{r, include=FALSE}```.  There should be no "setup" tag in your submission

File: edav_info_heatmap_korean.Rmd [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## R Markdown
+16.1 개요
```
Comment (st3186) [inline edav_info_heatmap_korean.Rmd]: Note that multiple hashtags like "## R Markdown" above will be rendered as sub-items (like, 4.1 R Markdown).  Given that, did you mean to write 16.1,2,3,etc.?  You may prefer to replace that with ## 개요, ## tl;dr, etc.

**Review by st3186 — CHANGES_REQUESTED** [review]
The Rmd file looks good except for the use of the named `setup` chunk, and maybe the use of numbered sections (which will be subject to change and will not update automatically if written this way).

Also, this should be merged into jtr13:master from your own new branch, not your own master.

**Comment by st3186** [comment]
Hi! I started making some necessary changes to your branch before I realized that you're still trying to do a pull request from your master branch into jtr13:master.  The pull request actually cannot be master->master; can you add your .rmd file to a new branch and try to create a pull request from there?  Thanks!

**Comment by st3186** [comment]
Hi Julia, just checking I'm not sure that what you did worked.  Do you mind creating a brand new branch with just your .Rmd file and then trying to make pull request from that branch into jtr13:master? I think you closed and reopened this pull request but I'm not sure how to make it work without doing what I said above.  Sorry about that and thank you!

**Comment by skyetim** [comment]
Duplicate of #171

**Pull request closed without merging**